# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLContextValidate.cpp
# Commit: e030be1d2dd9
# Full Hash: e030be1d2dd9b0bb47680fac29f20447c9d66f8c
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-02-21 09:51:10
# Regressor Bug: 1616715
# File Overlap Count: 1
# Description:
#   Bug 1616715 - Make ReadPixelDesc per-call. r=lsalzman
#   
#   GetSurfaceSnapshot (and similar) want to ReadPixels the backbuffer,
#   without having to navigate the GL PIXEL_PACK state.
#   Since this is cold cold code, it's nice to reduce the backend state by
# ==============================================================================

diff -r adc841b3e204 -r e030be1d2dd9 dom/canvas/WebGLContextValidate.cpp
--- a/dom/canvas/WebGLContextValidate.cpp	Fri Feb 21 02:27:08 2020 +0000
+++ b/dom/canvas/WebGLContextValidate.cpp	Fri Feb 21 03:12:41 2020 +0000
@@ -589,10 +589,6 @@
   mPixelStore.mUnpackSkipRows = 0;
   mPixelStore.mUnpackSkipPixels = 0;
   mPixelStore.mUnpackAlignment = 4;
-  mPixelStore.mPackRowLength = 0;
-  mPixelStore.mPackSkipRows = 0;
-  mPixelStore.mPackSkipPixels = 0;
-  mPixelStore.mPackAlignment = 4;
 
   mPrimRestartTypeBytes = 0;
 